packages/typegpu/tests/tgsl/wgslGenerator.test.ts

```diff
@@ -103,10 +103,10 @@ describe('wgslGenerator', () => {
 
     expect(parsedBody).toStrictEqual([
       NODE.block,
-      Object.entries(literals).map(([key, { value }]) => [
+      Object.entries(literals).map(([key, { wgsl }]) => [
```

Comment:
Since we're testing `tinyest` here, we should expect it to equal `value`, not `wgsl`.
```suggestion
      Object.entries(literals).map(([key, { value }]) => [
```